moveit_ros/move_group/src/move_group_capability.cpp

```diff
@@ -34,6 +34,7 @@
 
 /* Author: Ioan Sucan */
 
+#include <moveit/utils/moveit_error_code.h>
 #include <moveit/moveit_cpp/moveit_cpp.h>
 #include <moveit/move_group/move_group_capability.h>
 #include <moveit/robot_state/conversions.h>
```

Comment:
nit: alphabetize

Reply:
I shifted the newly introduced include, but I won't manually sort all existing ones.